JavaScriptSPA/ui.js

```diff
@@ -11,14 +12,13 @@ function showWelcomeMessage(account) {
     // Reconfiguring DOM elements
     cardDiv.style.display = 'initial';
     welcomeDiv.innerHTML = `Welcome ${account.name}`;
-    signInButton.nextElementSibling.style.display = 'none';
-    signInButton.setAttribute("onclick", "signOut();");
-    signInButton.setAttribute('class', "btn btn-success")
-    signInButton.innerHTML = "Sign Out";
+    signInButton.style.display = 'none';
+    signOutButton.style.display = 'initial';
 }
 
 function updateUI(data, endpoint) {
   console.log('Graph API responded at: ' + new Date().toString());
+  console.log(data)
```

Comment:
I would say any time you console log something in this sample, add a label so its clear what the data is.

Reply:
My bad, I forgot this. Removing...